src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/extensions */
+
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  switch (operation) {
+    case '÷':
+      return num1 / num2;
```

Comment:
If `num2` is zero(0) this will throw a `DivisionByZero` error.
Kindly cover this.